README.md

```diff
@@ -25,7 +25,8 @@ OpenCensus Go libraries require Go 1.8 or later.
 ## Exporters
 
 OpenCensus can export instrumentation data to various backends. 
-Currently, OpenCensus supports:
+OpenCensus has exporter implementation for the following, users
+can implement their own exporters by implementing the exporter interfaces:
```

Comment:
Want to link "exporter interfaces" to the godoc?

Reply:
Done.